assets/js/phoenix_live_view.js

```diff
@@ -560,18 +560,17 @@ export let Browser = {
     req.onerror = () => callback(400)
     req.ontimeout = () => callback(504)
     req.onreadystatechange = () => {
-      if(req.readyState !== 4){ return } 
+      if(req.readyState !== 4){ return }
       if(req.getResponseHeader(LINK_HEADER) !== "live-link"){ return callback(400) }
       if(req.status !== 200){ return callback(req.status) }
       callback(200, req.responseText)
     }
     req.send()
   },
 
-  pushState(kind, meta, to, callback){ 
+  pushState(kind, meta, to){
```

Comment:
Callback was unused.

Reply:
It'll be great if we can expose a public API for adding a callback here. This is useful when integrating with google analytics which doesn't fire for single application type-like behaviors.